jck/runtime.api/playlist.xml

```diff
@@ -387,6 +387,11 @@
 			<comment>Disabled on osx for infrastructure/issues/5254, and on rest for backlog/issues/485. Awaiting some automation improvements, in the interim these targets will be run manually</comment>
 			<version>11</version>
 		</disabled>
+		<disabled>
+			<comment>Disabled on osx for infrastructure/issues/5254. Awaiting some automation improvements, in the interim these targets will be run manually</comment>
+			<plat>x86-64_mac(_mixed)?</plat>
```

Comment:
All builds are switched (except zos). https://github.com/adoptium/aqa-tests/issues/1728
`_mixed` can be removed.

Reply:
Removed `_mixed`. Please review